qemu/tests/cfg/netuser_buffer_test.cfg

```diff
@@ -0,0 +1,10 @@
+- netuser_buffer_test:
+    only Linux
+    only virtio_net
+    virt_test_type = qemu
+    backup_image_before_testing = yes
```

Comment:
Seems you forget the restore one.

Reply:
Sorry, it was my mistake, I will add it in the next update.